scalability_and_performance/ztp_far_edge/ztp-sno-additional-worker-node.adoc

```diff
@@ -1,12 +1,25 @@
 :_content-type: ASSEMBLY
 [id="ztp-sno-additional-worker-node"]
-= Adding worker nodes to {sno} clusters with GitOps ZTP
+= Expanding {sno} clusters with GitOps ZTP
 include::_attributes/common-attributes.adoc[]
 :context: sno-additional-worker
 
 toc::[]
 
-You can add one or more worker nodes to an existing {sno} cluster with GitOps ZTP. Adding worker nodes does not require any downtime for the existing {sno}.
+You can expand {sno} clusters with GitOps ZTP. When you add worker nodes to {sno} clusters, the original {sno} cluster retains the control plane node role. Adding worker nodes does not require any downtime for the existing {sno} cluster.
+
+[NOTE]
+====
+Although there is no specified limit on the number of worker nodes that you can add to a {sno} cluster, you must revaluate the reserved CPU allocation on the control plane node for the additional worker nodes.
+====
+
+If you require workload partitioning on the worker node, you must deploy and remediate the managed cluster policies on the hub cluster before installing the node. This way, the workload partitioning `MachineConfig` objects are rendered and associated with the `worker` `MachineConfig` pool before the GitOps ZTP workflow applies the `MachineConfig` ignition file to the worker node.
```

Comment:
You can s/`worker` `MachineConfig` pool/`worker` machine config pool
It might look a little better.